frontend/src/metabase/admin/people/selectors.js

```diff
@@ -31,3 +34,11 @@ export const getUserMemberships = createSelector(
 
 export const getUserTemporaryPassword = (state, props) =>
   state.admin.people.temporaryPasswords[props.userId];
+
+export const shouldNudgeToPro = createSelector(
```

Comment:
This is really neat! We definitely want to use redux-toolkit utilities whenever possible, it makes our redux code much cleaner.